### ISSUE TITLE
Missing swanky jar in ensime-server-jars list in .ensime

### ISSUE THREAD

**Issue by iht** [ensime/ensime-sbt#374]
Hi,

I am having a problem trying to use the .ensime file generated by ensime-sbt, with Emacs, using the latest unstable version of the Emacs package, and ensime-sbt 2.0.1. I am using SBT 1.0.3, and Scala 2.12.4. I am working on macOS High Sierra.

The project to reproduce the problem is available at https://github.com/iht/fpinscala

The problem is that when I try to launch ensime in Emacs, it fails because it cannot find the SwankFormat class. When I examine the .ensime generated using ensime-sbt, I am missing the following JAR in the ensimer-server-jars list (path is local to my machine):

```
"/Users/iht/.coursier/cache/v1/https/repo1.maven.org/maven2/org/ensime/swanky_2.12/2.0.0/swanky_2.12-2.0.0.jar"
````

I have checked with other .ensime files in some other of my projects, using other Scala and SBT versions, and that ensime-server-jar is always included.

When I add it manually to the .ensime file, Emacs works fine with Ensime.

### Steps to reproduce

1. Clone or checkout the repo https://github.com/iht/fpinscala
2. sbt ensimeConfig (this will use SBT 1.0.3 and Scala 2.12.4)
3. Check that the swanky JAR is missing in the .ensime file

### Expected behaviour

ensime-server-jars should contain also the path to the swanky JAR, matching the corresponding Scala and ensime-server versions (2.12 and 2.0.0, respectively, in this case).

Many thanks in advance for any help with this issue.

**Comment by yyadavalli** [comment]
I was unable to reproduce the issue, that jar was present in the `.ensime`, so try following the troubleshooting guide and let me know.
http://ensime.org/build_tools/sbt/#troubleshooting

**Comment by iht** [comment]
I have just done that, with the minor exception that I removed the subdirs under ~/.sbt/1.0, as I am using sbt 1.0.3. I entirely removed the ~/.ivy2 directory. I am using also coursier to download dependencies. I wiped out my entire ~/.coursier directory. And the JAR was back in the .ensime config file :)

So it was probably some kind of problem with my .coursier cache (that's where the Ensime JARs are located).

Thanks for the help.
